src/main/java/de/hpi/bpt/argos/storage/PersistenceAdapter.java

```diff
@@ -49,6 +49,14 @@
 	boolean createArtifact(PersistenceArtifact artifact, String fetchUri);
 
 	/**
+	 * This method saves a new event in the database and notifies connected web socket clients.
+	 * @param event - the created event
+	 * @param eventOwner - the entity, which owns the event
+	 * @return - true, if the event was stored in the database
+	 */
+	boolean createEvent(Event event, Entity eventOwner, String fetchUri);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Expected @param tag for 'fetchUri'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadocMethod_BP2016)